### DISCUSSION TITLE
Way to model color coded/keyed connectors

### DISCUSSION THREAD

**Discussion by KaricAllegra** [netbox-community/netbox#17454] [Ideas]
As far as I could find in my searching, there is currently no way to model a cable with one color and the connectors/boots as another (without using custom fields). If there is, I apologize for missing it.

I suggest a way to model this, as it is critical for designing and documenting secure networks. In my world, security classifications are often color-coded with a certain cable jacket color and a different connector or boot color. They may also use a different type of keyed connector to ensure no cross-patching is possible.

My idea is that at cable creation, there could be two additional optional input boxes: one that allows a user to choose a boot/connector color that differs from the cable color and another that allows for a Keyed yes/no choice. If these are not chosen, then the default cable behavior will be the same as it currently is.

**Comment by ghost** [comment]
that's an interesting request. you should submit a FR for this. It's possible it will make it in because I can certainly see the use case.

**Reply by CADbloke** [reply]
submitted https://github.com/netbox-community/netbox/issues/19636
